packages/lib-classifier/src/plugins/tasks/TextTask/models/TextPreviousAnnotations.js

```diff
@@ -0,0 +1,10 @@
+import { types } from 'mobx-state-tree'
+
+const TextPreviousAnnotations = types.model('TextPreviousAnnotations', {
+  id: types.identifier,
+  taskKey: types.string,
+  taskType: types.string,
```

Comment:
Can `id`, `taskKey` and `taskType` be composed from the `Annotation` model?

Reply:
Looking over the `Annotation` model, there are the snapshot processors, a couple of computed views, and an action that this model doesn't need, so I'm going to forgo the composition.